doc/devel/documenting_mpl.rst

```diff
@@ -492,11 +308,91 @@ For everything but top level chapters, please use ``Upper lower`` for
 section titles, e.g., ``Possible hangups`` rather than ``Possible
 Hangups``
 
+Setters, getters, and keyword arguments
+=======================================
```

Comment:
Why is this way down here instead of up w/ the numpydoc section?

Reply:
Moved back up